geodiff/src/geodiff.h

```diff
@@ -41,21 +41,47 @@ extern "C" {
 #endif
 
 /**
-* NOTE:
-* the messages printed to stdout can be controlled by
-* environment variable GEODIFF_LOGGER_LEVEL
-* GEODIFF_LOGGER_LEVEL = 0 nothing is printed
-* GEODIFF_LOGGER_LEVEL = 1 errors are printed
-* GEODIFF_LOGGER_LEVEL = 2 errors and warnings are printed
-* GEODIFF_LOGGER_LEVEL = 3 errors, warnings and infos are printed
-* GEODIFF_LOGGER_LEVEL = 4 errors, warnings, infos, debug messages are printed
+ * Initialize library
+ *
+ * The default logger is set, where all the messages printed are printed to stdout and
+ * can be controlled by environment variable GEODIFF_LOGGER_LEVEL
+ * GEODIFF_LOGGER_LEVEL = 0 nothing is printed
+ * GEODIFF_LOGGER_LEVEL = 1 errors are printed
+ * GEODIFF_LOGGER_LEVEL = 2 errors and warnings are printed
+ * GEODIFF_LOGGER_LEVEL = 3 errors, warnings and infos are printed
+ * GEODIFF_LOGGER_LEVEL = 4 errors, warnings, infos, debug messages are printed
+ */
+GEODIFF_EXPORT void GEODIFF_init();
+
+/**
+* Type of message level to log
 */
+enum LoggerLevel
+{
+  LevelError = 0,
+  LevelWarning = 1,
+  LevelInfo = 2,
+  LevelDebug = 3
+};
```

Comment:
it would be good to give it GEODIFF prefix (and also to LoggerCallback) to avoid possible namespace clashes with other headers...